app/templates/components/events/view/overview/event-sponsors.hbs

```diff
@@ -20,7 +20,7 @@
         <td>ABC</td>
         <td>1</td>
         <td>
-          <div class="ui horizontal compact basic buttons">
+          <div class="ui vertical compact basic buttons">
```

Comment:
Here, why I have replaced vertical from horizontal? because
- It looks good and doesn't take space vertically because we have an image in the table which has 100*100 dimensions so it does distort the UX.
- If I use {{if device.isTablet 'vertical'}} this will not solve the problem for small tablet landscape and the problem is occurring only in small tablet landscape and tablet portrait.
- If I make a change only in the table by adding compact and let this remain horizontal then the problem of overflow is not resolved.

@niranjan94 @CosmicCoder96 @geekyd @sumedh123 @anu-007 @utkarshnath Please review. Tha